docs/schema_guide.md

```diff
@@ -1,157 +1,29 @@
 schema defines some required fields, requirements about feature labels, and some optional fields that mostly help with
 presentation.
 
-The number of fields is rather low, and we expect that information needed to populate those fields should either already
-be present in datasets prepared by a submitter or be easy to obtain. However, this still leaves the task of actually
-manipulating the dataset so that it follows the schema: adjusting field names, ensuring proper ontologies are used,
-converting gene symbols to a common set, etc. This can be tedious and error-prone, and at the beginning of the hosted
-cellxgene project, this was always done with engineering support. As we increase the rate at which we add data, we want
-to eliminate the need for engineering support so that ultimately submitters themselves can create files that follow the
-schema.
+## `cellxgene-schema validate`
 
-## `cellxgene-schema apply`
+To enable validation of datasets, we have a python package `cellxgene-schema` that supplies the `validate` command, which validates that a given h5ad follows a version of the schema.
 
-To enable this, we have a python package the supplies the `cellxgene-schema` command that handles applying and verifying the schema.
-Its first subcommand, `cellxgene-schema apply`, takes three inputs:
-
-1. A source h5ad file. The input needs to be an AnnData file, so if a submitter has, say, a serialized Seurat or
-   SingleCellExperiment object, it needs to be converted to AnnData first. This can be done with
-   [sceasy](https://github.com/cellgeni/sceasy) or via
-   [Seurat](https://satijalab.org/seurat/v3.1/conversion_vignette.html).
-2. A configuration yaml file that describes the fields to add and conversions to apply (see below).
-3. A name for the new h5ad file that should follow the schema.
-
-### Configuration yaml
-
-The configuration yaml file describes how to apply the schema. This is an example of a "skeleton" yaml that has all the
-fields required for the 1.1.0 schema but is not yet filled in with any logic:
+The usage of this command for a data contributor should be as follows:
 
 ```
-uns:
-    version:
-        corpora_schema_version: 1.1.0
-        corpora_encoding_version: 0.1.0
-    title:
-    layer_descriptions:
-    organism_ontology_term_id:
-obs:
-    tissue_ontology_term_id:
-    assay_ontology_term_id:
-    disease_ontology_term_id:
-    cell_type_ontology_term_id:
-    sex:
-    ethnicity_ontology_term_id:
-    development_stage_ontology_term_id:
-fixup_gene_symbols:
+cellxgene-schema validate input.h5ad
 ```
 
-#### Unstructured metadata
-The first section is `uns`, which includes metadata fields that describe the whole dataset (see 
-[here](https://anndata.readthedocs.io/en/latest/) for further description of `uns` and `obs`.).
+If the validation succeeds, the command will have a zero exit code. If it does not, it will have a non-zero exit code and will print validation failure messages.
```

Comment:
If the validation succeeds, the command returns a zero exit code; otherwise,  it returns a non-zero exit code and prints validation failure messages.